distribution/pom.xml

```diff
@@ -33,20 +33,6 @@
               <appendAssemblyId>false</appendAssemblyId>
             </configuration>
           </execution>
-          <!-- execution>
```

Comment:
Just some basic pom clean up here for services, even though hoot-josm doesn't make use of anything in hoot-services.